public/data/eddiejaoude.json

```diff
@@ -43,6 +43,11 @@
       "name": "Listen to my Podcast",
       "url": "https://anchor.fm/eddiejaoude",
       "icon": "codeforces"
+    },
+    {
+      "name": "Donate to me on GitHub Sponsors",
+      "url": "https://github.com/sponsors/eddiejaoude",
+      "icon": "github"
```

Comment:
```suggestion
      "icon": "dollar"
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Great, thank you 👍